environment.yml

```diff
@@ -13,22 +13,25 @@ dependencies:
     - packaging
```

Comment:
Line 7: can we remove `pip` from the list?

Reply:
We could, but then people would get a warning that says: "Warning : you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies...". See https://stackoverflow.com/questions/58544099/warning-after-i-run-the-command-conda-env-create-f-environment-yml